### PR TITLE
Issue #2: Loan Dashboard

### PR DESCRIPTION
**Pull request by archanglmr**
Issue #2 

Overall cleaned up some of the base stuff that we won't need in this project (plenty more to do), specifically extra bootstrap styles and removing NX as a suggested VS Code plugin since this isn't an NX repo.

The rest is really the Verifications Dashboard using files as data sources. Observables are used in the component so I can just swap the BehaviorSubjects with the observable of a future service.

No store or routing implemented yet. Figure this was a good point to show and will add the store when more interaction starts.

Pay attention to where some of the modified CSS is. There are so many top(ish) level spots but I think I have about the right area. Please correct me if my changes are in the wrong place.

When you're happy with the PR please "squash and merge" and we can use the PR title as the commit message.

PS: ignore the branch name, I was going to do the Verification Dashboard first but realized I needed a way to get there so changed midcourse.

### CODE REVIEW

File: src/app/components/masterpage/header/header.component.ts [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-import { Component, OnInit, signal } from '@angular/core';
+import { ChangeDetectionStrategy, Component, HostBinding, OnInit, signal } from '@angular/core';
 import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
 import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-header',
   styleUrls: ['./header.component.scss'],
   templateUrl: './header.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
```
Comment (JerrolKrause) [inline src/app/components/masterpage/header/header.component.ts:10]: Nice catch, need to update the source template

File: src/app/components/masterpage/masterpage/masterpage.component.html
Comment (JerrolKrause) [inline src/app/components/masterpage/masterpage/masterpage.component.html:1]: Double nested folder structure? IE masterpage/masterpage?

File: src/app/routes/home/components/help-panel/help-panel.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<p-card>
+  <h2>Need Help?</h2>
+  <p>
+    <a href="tel:+{{ phoneNumber$ | async }}">Call us</a> at <a href="tel:+{{ phoneNumber$ | async }}">{{ phoneNumber$ | async | phoneNumber }}</a>
```
Comment (JerrolKrause) [inline src/app/routes/home/components/help-panel/help-panel.component.html:4]: I'd suggest using a single subscription, very minor

> <a *ngIf="phoneNumber$ | async as phoneNumber" href="tel:+{{ phoneNumber }}"></a>

etc etc

File: src/app/routes/home/components/loan-thumb/loan-thumb.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { ChangeDetectionStrategy, Component, input } from '@angular/core';
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { Models } from '../../../../shared/models';
```
Comment (JerrolKrause) [inline src/app/routes/home/components/loan-thumb/loan-thumb.component.ts:3]: Minor, import models from the "$models" path.

File: src/app/routes/no-content/no-content.component.ts [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
     <app-masterpage>
-      <div class="container mt-3">
+      <div>
         <h1>404: page missing</h1>
         <p><a routerLink="/">Click here to go to the homepage.</a></p>
       </div>
     </app-masterpage>
   `,
+  styleUrls: ['./no-content.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
```
Comment (JerrolKrause) [inline src/app/routes/no-content/no-content.component.ts:14]: Nice catch, I need to update the source template

File: src/app/shared/models/global.models.ts
Comment (JerrolKrause) [inline src/app/shared/models/global.models.ts:1]: Very nice!

File: src/assets/mock-data/loans.ts
Comment (JerrolKrause) [inline src/assets/mock-data/loans.ts:1]: Good approach, I like it

**Review by JerrolKrause — APPROVED** [review]

File: src/app/routes/home/components/loan-thumb/loan-thumb.component.ts (hunk 3 above)
Comment (archanglmr) [reply src/app/routes/home/components/loan-thumb/loan-thumb.component.ts:3]: Guessing you mean `import { Models } from '$shared'`

**Pull request merged**